itou/job_applications/models.py

```diff
@@ -375,11 +377,19 @@ def eligible_as_employee_record(self, siae):
             self._eligible_job_applications_without_employee_record(siae),
             self._eligible_job_applications_with_a_suspended_or_extended_approval(siae),
         )
+        approvals_to_exclude = (
+            EmployeeRecord.objects.filter(
+                asp_id=siae.convention.asp_id,
+            )
+            .exclude(status__in=[employeerecord_enums.Status.NEW])
+            .values("approval_number")
+        )
```

Comment:
Si je reformule, on exclus les PASS qui on déjà une fiche en NEW pour la SIAE cible.
Ça ne rentre pas en conflit avec `_eligible_job_applications_with_employee_record` ?
En fait, ça semble son antithèse non ?

Reply:
Oui, on les exclus ici afin de ne pas les exclure dans la requête d'après :exploding_head:, sinon comme tu le dis ça annule ce que nous renvois `_eligible_job_applications_with_employee_record()`.

J'ai rajouté un commentaire :
```
# We need to exclude NEW employee records otherwise we are shooting ourselves in the foot by excluding
# job applications selected in `._eligible_job_applications_with_employee_record()`
.exclude(status__in=[employeerecord_enums.Status.NEW]).values("approval_number")
```